libraries/standard/http/src/http_client.c

```diff
@@ -541,6 +542,11 @@ HTTPStatus_t HTTPClient_Send( const HTTPTransportInterface_t * pTransport,
         IotLogError( "Parameter check failed: pRequestHeaders->pBuffer is NULL." );
         returnStatus = HTTP_INVALID_PARAMETER;
     }
+    else if( pResponse && ( pResponse->pBuffer == NULL ) )
```

Comment:
(Nit): If one operand is being _explicitly_ checked against `NULL`, then the other operand in the expression should too.
`if ( ( pResponse != NULL ) && ( pResponse->pBuffer == NULL ) )`

Reply:
sure